### DISCUSSION TITLE
Enhancing Elementary OS with a System Tray Icons.

### DISCUSSION THREAD

**Discussion by Mashrafi2006** [elementary/triage#413] [Ideas]
Adding a System Tray Icon to the panel in Elementary OS can enhance the user experience by providing quick access to frequently used applications and settings. This can save time and improve productivity. Plus, the System Tray Icon can be enabled or disabled in the settings, giving users control over their experience.

**Comment by danirabbit** [comment]
Hey thanks for your suggestion. There is no standard "system tray" API in freedesktop, so this is not something that makes sense for us to pursue.

There are supported APIs for things like application backgrounding, quick action launching, notifications, media controls, screen casting, location, etc. App developers should be writing their apps against these commonly supported and agreed upon cross desktop APIs and Portals.
